### PR TITLE
Promote this repo to the "official" status

### PR DESCRIPTION
**Pull request by dvitas**
Hi team!
Please ACK / NACK.

Changes in comments and README only, no actual code change.
README.md is basically the home Wiki page wirh the different links format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
- https://sourceforge.net/projects/mstpd/
-
-And the original author is: Vitalii Demianets <dvitasgs@gmail.com>
-
-Description
------------
-
-**msptd** implements the IEEE 802.1s Multiple Spanning Tree Protocol.
+MSTPD is an open source user-space daemon licensed under GPLv2.
+At the moment the latest version 0.04 is reported to be compliant with IXIA ANVL RSTP test suite, with the notable exception of looped-back BPDUs (see discussion on the matter on the Implementation Features wiki page: https://github.com/mstpd/mstpd/wiki/ImplementationFeatures).
```
Comment (vivien) [inline README.md]: We don't have a 0.04 Git tag yet. Nor is 0.03 tagged. Should we drop the version number here?

**Comment by vivien** [comment]
Hi @dvitas, I registered the #mstpd IRC channel on freenode, with notifications from Github ;-)

In addition to the comment I wrote above, it'd be great to mention it in the readme, if you think that it is a good idea.

**Comment by dvitas** [comment]
In general I agree that version number in this piece of text is not strictly necessary. Also, as documentation usually tends to be outdated, it will be misleading sooner or later, and not updated in time. So I agree to remove it from here.
As for the tags - we should definitely start to create them. Right from the next version, as soon as issue #12 is resolved.
As for the IRC channel - yes, it is good to mention it in the Readme. I am currently preparing a new pull request here: https://github.com/dvitas/mstpd/commits/devel . Can you please send a patch against this branch which shows how exactly you want the IRC channel to be mentioned in Readme? Then I will create a new pull request.

**Comment by vivien** [comment]
Do we want to tag 0.03 or is it unecessary?
For the IRC mention in the readme, I don't mind, really. I'd go with a simple sentence like: *Feel free to join and chat with us on the #mstpd @ freenode!"

**Comment by dvitas** [comment]
Personally I see no reason to add historical tags. Let's start adding them from the next version (0.05) on. YMMV.

I have created a new PR #13 with additional commit and all the discussed changes incorporated, so I close this one.

**Pull request closed without merging**